CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -1043,7 +1046,7 @@ protected function postProcessPremium($premiumParams, $contribution) {
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
-  public static function processFormContribution(
+  protected static function processFormContribution(
```

Comment:
Obviously it's not supported to call this from outside core - but I did do a universe search to double check